templates/default.yaml

```diff
@@ -123,6 +123,19 @@ mountInotify: null
 # ADVANCED CONFIGURATION
 # ===================================================================== #
 
+# User to be used inside the VM
+user:
+  # User name. An explicitly specified username is not validated by Lima.
+  # 🟢 Builtin default: same as the host username, if it is a valid Linux username, otherwise "lima"
+  username: null
+  # Numeric user id. It is not currently possible to specify a group id.
+  # 🟢 Builtin default: same as the host user id of the current user (NOT a lookup of the specified "username").
+  uid: null
+  # Home directory inside the VM, NOT the mounted home directory of the host.
+  # It can use the following template variables: {{.Name}}, {{.Hostname}}, {{.UID}}, {{.User}}, and {{.Param.Key}}.
+  # 🟢 Builtin default: "/home/{{.User}}.linux"
```

Comment:
Should this be using {{.OS}} perhaps? (for Linux, but lowercase the value). Only makes a difference for FreeBSD

Reply:
Yeah, not really part of this PR.